### PR TITLE
Update .gitignore

### PR DESCRIPTION
**Pull request by mferenca**
@vkaracic - Please take a look

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
-# Custom ignore rules:
-## Python general
```
Comment (vkaracic) [inline .gitignore:2]: Why are you removing this?

File: .gitignore [hunk 2]
```diff
@@ -1,3 +1,4 @@
...
 *.py[cod]
+
+# Packages
+*.egg-info
```
Comment (vkaracic) [inline .gitignore]: Add a new line, please.

**Review by vkaracic — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (mferenca) [reply .gitignore:2]: It doesn't look right.

File: .gitignore (hunk 2 above)
Comment (mferenca) [reply .gitignore]: Sure.

File: .gitignore (hunk 1 above)
Comment (vkaracic) [reply .gitignore:2]: How so?

Comment (mferenca) [reply .gitignore:2]: Looks better without it.

Comment (vkaracic) [reply .gitignore:2]: You should leave at least one header comment explaining the rules below.
See how it's done here: https://github.com/github/gitignore/blob/master/Python.gitignore

Comment (mferenca) [reply .gitignore:2]: I prefer following style: https://github.com/edx/edx-ora2/blob/master/.gitignore
But if you really insist I'll add header comment.

**Review by vkaracic — APPROVED** [review]

**Pull request merged**